internal/docstore/dynamodocstore/dynamo.go

```diff
@@ -540,4 +540,5 @@ var errorCodeMap = map[string]gcerrors.ErrorCode{
 	dyn.ErrCodeTransactionCanceledException:             gcerr.FailedPrecondition,
 	dyn.ErrCodeTransactionInProgressException:           gcerr.InvalidArgument,
 	dyn.ErrCodeIdempotentParameterMismatchException:     gcerr.InvalidArgument,
+	"ValidationException":                               gcerr.InvalidArgument,
```

Comment:
Weird they don't expose the type for dynamo and this seems a pretty common error.

Reply:
Agreed. I grepped the package tree and didn't see it.